cuegui/cuegui/JobMonitorTree.py

```diff
@@ -435,7 +435,7 @@ def _processUpdate(self, work, rpcObjects):
                 if proxy in self.__userColors:
                     self._items[proxy].setUserColor(self.__userColors[proxy])
 
-            self.verticalScrollBar().setValue(scrolled)
+            self.verticalScrollBar().setRange(scrolled, len(jobObjects.keys() - scrolled))
```

Comment:
Lint check is failing with:
```
cuegui/JobMonitorTree.py:438:60: E0602: Undefined variable 'jobObjects' (undefined-variable)
```

Should that be `rpcObjects`?

Reply:
You are correct, thanks for spotting that (was an older change I was porting over and missed that!) Thanks so much. Also I missed the line you highlighted in the logs (thanks again for that). I'll update the code